lib/Cro/HTTP/Client/CookieJar.pm6

```diff
@@ -49,6 +49,31 @@ monitor Cro::HTTP::Client::CookieJar {
         False
     }
 
+    method !get-cookie-lifetime(Cro::HTTP::Cookie $cookie, CookieState $state) {
+      if $cookie.max-age.defined {
+          $state.persistent = True;
```

Comment:
Just noticed, but the added code uses 2-space indents, while the rest of the Cro code uses 4-space indents.